Formula/v/vulkan-extensionlayer.rb

```diff
@@ -56,15 +57,16 @@ def caveats
   test do
     ENV.prepend_path "VK_LAYER_PATH", share/"vulkan/explicit_layer.d"
     ENV["VK_ICD_FILENAMES"] = Formula["vulkan-tools"].lib/"mock_icd/VkICD_mock_icd.json"
+    ENV[OS.mac? ? "DYLD_LIBRARY_PATH" : "LD_LIBRARY_PATH"] =
+      Formula["vulkan-loader"].lib.to_s
```

Comment:
Needing this looks suspicious. Seems like we should add this path to the binary rpaths instead?

Reply:
@carlocab How would we do that?  I'm not familiar enough with that side of things.

Reply:
Try passing
```
-DCMAKE_INSTALL_RPATH=#{rpath(target: Formula["vulkan-loader"].opt_lib)}
```
to the first CMake invocation. Hopefully that will just work. If not we can try to tweak the `source:` argument of `#rpath`. If that doesn't work either we'll need to take a closer look at the CMake scripts.